src/StreetNameRegistry/Municipality/Events/StreetNameWasProposedForMunicipalityMerger.cs

```diff
@@ -11,7 +11,7 @@ namespace StreetNameRegistry.Municipality.Events
 
     [EventTags(EventTag.For.Sync)]
     [EventName(EventName)]
-    [EventDescription("De straatnaam werd voorgesteld voor een gemeente fusie.")]
+    [EventDescription("De straatnaam werd voorgesteld ter voorbereiding van een gemeentefusie.")]
     public sealed class StreetNameWasProposedForMunicipalityMerger : IMunicipalityEvent, IHasPersistentLocalId
     {
```

Comment:
I've left the property description of StreetNameNames and HomonymAdditions untouched although they differ from what is mentioned in the sprint review PPT. 
![image](https://github.com/user-attachments/assets/d575c821-9a28-48e5-b1da-99e6fa79e1d8)